gensim/test/test_fasttext.py

```diff
@@ -448,27 +429,21 @@ def test_sg_neg_training(self):
     def test_online_learning(self):
         model_hs = FT_gensim(sentences, size=10, min_count=1, seed=42, hs=1, negative=0)
         self.assertTrue(len(model_hs.wv.vocab), 12)
-        self.assertTrue(len(model_hs.wv.ngrams), 202)
         self.assertTrue(model_hs.wv.vocab['graph'].count, 3)
-        self.assertFalse('tif' in model_hs.wv.ngrams)
         model_hs.build_vocab(new_sentences, update=True)  # update vocab
         self.assertEqual(len(model_hs.wv.vocab), 14)
-        self.assertTrue(len(model_hs.wv.ngrams), 271)
         self.assertTrue(model_hs.wv.vocab['graph'].count, 4)
         self.assertTrue(model_hs.wv.vocab['artificial'].count, 4)
-        self.assertTrue('tif' in model_hs.wv.ngrams)  # ngram added because of the word `artificial`
 
     def test_online_learning_after_save(self):
         tmpf = get_tmpfile('gensim_fasttext.tst')
         model_neg = FT_gensim(sentences, size=10, min_count=0, seed=42, hs=0, negative=5)
         model_neg.save(tmpf)
         model_neg = FT_gensim.load(tmpf)
         self.assertTrue(len(model_neg.wv.vocab), 12)
-        self.assertTrue(len(model_neg.wv.ngrams), 202)
         model_neg.build_vocab(new_sentences, update=True)  # update vocab
         model_neg.train(new_sentences, total_examples=model_neg.corpus_count, epochs=model_neg.iter)
         self.assertEqual(len(model_neg.wv.vocab), 14)
-        self.assertTrue(len(model_neg.wv.ngrams), 271)
```

Comment:
Why you removed part of tests (I mean all removed lines in tests)?

Reply:
As I mentioned in the PR, one optimization was to remove the storage of ngrams on the model and solely rely on the hashes. This is why any tests that assert the number of ngrams in the model are no longer neccesary.

With the lack of ngrams, the `__contains__` check now also only uses the hashed and bucketed ngrams, which is why a 'real' OOV is a lot rarer (i.e. it will only happen if not all buckets are occupied and none of the ngrams in the token match any occupied bucket).